cclib/parser/molcasparser.py

```diff
@@ -0,0 +1,181 @@
+        #                     **** Cartesian Coordinates / Bohr, Angstrom **** 
+        #                     ************************************************ 
+
+        #      Center  Label                x              y              z                     x              y              z
+        #         1      C1               0.526628      -2.582937       0.000000              0.278679      -1.366832       0.000000
+        #         2      C2               2.500165      -0.834760       0.000000              1.323030      -0.441736       0.000000
+        if line[25:63] == 'Cartesian Coordinates / Bohr, Angstrom':
+            if not hasattr(self, 'atomnos'):
+                self.atomnos = []
+            if not hasattr(self, 'atomcoords'):
+                self.atomcoords = []
+
+            self.skip_lines(inputfile, ['stars', 'blank', 'header'])
+
+            line = next(inputfile)
+
+            atomelements = []
+            atomcoords = []
+
+            while not self.re_dashes_and_spaces.search(line):
+                sline = line.split()
+                atomelements.append(self.re_atomelement.search(sline[1]).groups()[0])
+                atomcoords.append(list(map(float, sline[5:])))
+                line = next(inputfile)
+
+            self.atomcoords.append(atomcoords)
+
+            if self.atomnos == []:
+                self.atomnos = [utils.PeriodicTable().number[atomelement] for atomelement in atomelements]
+
+            if not hasattr(self, 'natom'):
+                self.set_attribute('natom', len(self.atomnos))
+
+        ## This section is present when executing &SCF.
+        if line[0:29] == '++    Orbital specifications:':
+
+            self.skip_lines(inputfile, ['dashes', 'blank'])
+
+            line = next(inputfile)
+
+            while line[0:2] != '--':
+
+                if line[6:30] == 'Total number of orbitals':
+                    self.set_attribute('nmo', int(line.split()[-1]))
+                if line[6:31] == 'Number of basis functions':
+                    self.set_attribute('nbasis', int(line.split()[-1]))
+
+                line = next(inputfile)
+
+        if line[6:23] == 'Molecular charge ':
+            self.set_attribute('charge', int(float(line.split()[-1])))
+
+        ## This section is present when executing &SCF.
+        if line[0:34] == '++    Optimization specifications:':
+
+            self.skip_lines(inputfile, ['dashes',
+                                        'blank',
+                                        'SCF Algorithm',
+                                        'density differences',
+                                        'Number of density matrices in core',
+                                        'Maximum number of NDDO SCF iterations',
+                                        'Maximum number of HF  SCF iterations'])
+
+            line = next(inputfile)
+            assert line[6:37] == 'Threshold for SCF energy change'
```

Comment:
This can be combined with the previous line, I presume it wasn't because you plan to actually parse this?